trollhunter-api/core/TrollHunterApi/Controllers/FeedController.cs

```diff
@@ -0,0 +1,30 @@
+using Microsoft.AspNetCore.Mvc;
```

Comment:
Can you add the Google Apache 2 license headers to the top of these files.

Reply:
Done.